packages/good-design/src/utils/formatVerifiedValues.ts

```diff
@@ -0,0 +1,12 @@
+export const formatVerifiedValues = ({ credentialSubject, typeName }: any) => {
+  if (!credentialSubject) return "Unverified";
+
+  switch (typeName) {
+    case "Age":
```

Comment:
don't you have enum for that ?

Reply:
if not please define

Reply:
age.min / age.max is not easily defined our map through an enum
I now tried it like this: https://github.com/GoodDollar/GoodWeb3-Mono/pull/136/commits/09c8cc6c23aa862858a16c0aa244543ed9cfea77